integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomainModelInPV.java

```diff
@@ -155,72 +193,116 @@ public static void initAll(@Namespaces(2) List<String> namespaces) {
     encryptionSecretName = "encryptionsecret";
     createSecretWithUsernamePassword(encryptionSecretName, domainNamespace, "weblogicenc", "weblogicenc");
 
     // create the PV and PVC to store application and model files
-    createPV(pvName, domainUid, "ItMiiDomainModelInPV");
-    createPVC(pvName, pvcName, domainUid, domainNamespace);
+    createPV(pvName, domainUid1, "ItMiiDomainModelInPV");
+    createPVC(pvName, pvcName, domainUid1, domainNamespace);
 
     // build the clusterview application
     Path distDir = buildApplication(Paths.get(APP_DIR, "clusterview"),
         null, null, "dist", domainNamespace);
     clusterViewAppPath = Paths.get(distDir.toString(), "clusterview.war");
     assertTrue(clusterViewAppPath.toFile().exists(), "Application archive is not available");
 
-    V1Pod webLogicPod = setupPVPod(domainNamespace);
+    logger.info("Setting up WebLogic pod to access PV");
+    V1Pod pvPod = setupWebLogicPod(domainNamespace);
 
+    ExecResult exec = null;
     try {
-      logger.info("Creating directory {0} in PV", "/shared/applications");
-      Exec.exec(webLogicPod, null, false, "/bin/sh", "-c", "mkdir /shared/applications");
+      logger.info("Creating directory {0} in PV", modelMountPath + "/applications");
+      exec = Exec.exec(pvPod, null, false, "/bin/sh", "-c", "mkdir -p " + modelMountPath + "/applications");
+      if (exec.stdout() != null) {
+        logger.info("Exec stdout {0}", exec.stdout());
+      }
+      if (exec.stderr() != null) {
+        logger.info("Exec stderr {0}", exec.stderr());
+      }
     } catch (IOException | ApiException | InterruptedException ex) {
       logger.warning(ex.getMessage());
     }
     try {
-      logger.info("Creating directory {0} in PV", "/shared/model");
-      Exec.exec(webLogicPod, null, false, "/bin/sh", "-c", "mkdir /shared/model");
+      logger.info("Creating directory {0} in PV", modelMountPath + "/model");
+      exec = Exec.exec(pvPod, null, false, "/bin/sh", "-c", "mkdir -p " + modelMountPath + "/model");
+      if (exec.stdout() != null) {
+        logger.info("Exec stdout {0}", exec.stdout());
+      }
+      if (exec.stderr() != null) {
+        logger.info("Exec stderr {0}", exec.stderr());
+      }
     } catch (IOException | ApiException | InterruptedException ex) {
       logger.warning(ex.getMessage());
     }
 
     try {
       //copy the model file to PV using the temp pod - we don't have access to PVROOT in Jenkins env
       logger.info("Copying model file {0} to pv directory {1}",
-          Paths.get(MODEL_DIR, modelFile).toString(), "/shared/model");
-      Kubernetes.copyFileToPod(domainNamespace, webLogicPod.getMetadata().getName(), null,
-          Paths.get(MODEL_DIR, modelFile), Paths.get("shared", "model", modelFile));
+          Paths.get(MODEL_DIR, modelFile).toString(), modelMountPath + "/model");
+      Kubernetes.copyFileToPod(domainNamespace, pvPod.getMetadata().getName(), null,
```

Comment:
replace with assertDoesNotThrow(() -> copyFileToPod()) and remove try, catch

Reply:
The copy command above some times generates exceptions which are harmless, If I put the copy command in assertDoesNotThrow its going to fail intermittently. With try catch I can simply ignore the exceptions.

Reply:
If exceptions are harmless, how about catching them in Kubernetes.copyFileToPod() method itself and ignore/log them?

Reply:
There are many IT files using the Kubernetes.copyFileToPod() method. I don't want to touch those files in this PR. Moved the copy command from test class to CommonTestUtils